server/graphql/invoice_line/src/mutations/outbound_shipment_line/line/insert.rs

```diff
@@ -56,7 +74,7 @@ pub fn insert(ctx: &Context<'_>, store_id: &str, input: InsertInput) -> Result<I
     map_response(
         service_provider
             .invoice_line_service
-            .insert_outbound_shipment_line(&service_context, input.to_domain()),
+            .insert_stock_out_line(&service_context, input.to_domain()),
```

Comment:
insert_out_invoice_line or InsertStockOutLine -> either way, would be good to align type and method name